app/mapper/CaseRequestMapper.scala

```diff
@@ -23,6 +23,25 @@ import pages._
 @Singleton
 class CaseRequestMapper {
 
+  def buildAgentDetails(answers: UserAnswers): Option[AgentDetails] = {
+    val whichBestDescribeYou: Option[WhichBestDescribesYou] = answers.get(WhichBestDescribesYouPage)
+
+    if (whichBestDescribeYou.get != WhichBestDescribesYou.BusinessRepresentative) { None }
+
+    val details: RegisterBusinessRepresenting = answers.get(RegisterBusinessRepresentingPage).get
+    Some(AgentDetails(
+      EORIDetails(
+        details.eoriNumber,
+        details.businessName,
+        details.addressLine1,
+        "", // Line 2 empty
+        "", // Line 3 empty
```

Comment:
We can probably remove these from the trader model as theyre always empty

Reply:
I would prefer to make a decision and refactor it in another PR. probably rename Line 1 as well